newswires/client/src/SideNav/SideNav.tsx

```diff
@@ -46,7 +49,8 @@ function decideLabelForQueryBadge(query: Query): string {
 }
 
 function presetName(presetId: string): string | undefined {
-	return presets.find((preset) => preset.id === presetId)?.name;
+	return [...presets, ...sportPresets].find((preset) => preset.id === presetId)
```

Comment:
this was a pass-by refactor, as I realised we weren't including the sports presets in the name mapping for search history query badges